mods/ra/maps/intervention/intervention.lua

```diff
@@ -207,6 +228,10 @@ Tick = function()
 			player.MarkFailedObjective(villageObjective)
 		end
 	end
+
+	local percentage = 100 - VillageHousesLeft() * 10
+	local color = GetColor(percentage)
+	UserInterface.SetMissionText(percentage .. "% of the village destroyed.", color)
 end
```

Comment:
You could move this into an `OnKilled` trigger set on every house in the village. Then this wouldn't need to run every tick.